### DISCUSSION TITLE
SITS version 1.5.0 - end-to-end EO data analysis using MPC

### DISCUSSION THREAD

**Discussion by gilbertocamara** [microsoft/PlanetaryComputer#341] [Show and tell]
Dear all, version 1.5.0 of R package `sits` is now on CRAN. `sits` is a TRL-9, operationally robust software that provides end-to-end support for big Earth observation data analysis of image time series using machine learning and deep learning methods. It provides access to SENTINEL-1-GRD, SENTINEL-1-RTC, SENTINEL-2-L2A and LANDSAT-C2-L2 collections in MPC. Highlights of version 1.5.0 include: 

(a) Production of regular EO data cubes from MPC collections;
(b) Merging Sentinel-1 and Sentinel-2 data cubes;
(c) State-of-the-art deep learning methods for image time series analysis, optimized for GPUs;
(d) Innovative self-organized maps (SOM) for quality assessment of training data sets;
(e) Bayesian post-processing methods to remove outliers from time series classification:
(f) Uncertainty map estimates of time series classification;
(g) Support for tuning deep learning models;
(h) Efficient performance of traditional ML models such as random forest and xgboost;
(i) Spatio-temporal segmentation for object-based image time series analysis;
(j) Accuracy assessment methods based on recommended best practices for LUCC map classification;
(k) Simple, user-friendly API that shortens the learning curve. 
(l) Detaliled documentation of the package is available  in an on-line book (https://e-sensing.github.io/sitsbook/). 

Support for `sits` is available in R processing environment of MPC.
